### ISSUE TITLE
Allow multiple exports

### ISSUE THREAD

**Issue by okelet** [Ddedalus/poetry-auto-export#2]
Hi, first of all, thanks for this plugin, I have been long waiting for this. I would suggest to add multiple export configurations. In our case, we have different requirements for regular building (no dev dependencies) and for testing (that includes dev dependencies).

This is our current structure, that could be similar to other projects:

```text
.
├── README.md
├── poetry.lock
├── pyproject.toml
├── src
│   ├── my_package
│   │   ├── __init__.py
│   │   ├── package_1
│   │   │   ├── __init__.py
│   │   │   └── app.py
│   └── requirements.txt
└── tests
    ├── requirements-dev.txt
    └── unit
        └── test_1.py
```

**Comment by Ddedalus** [comment]
Hi, I'm glad to hear you like the package. I think adding multiple export options shouldn't be too difficult. I'll try to have a look over the next few days...

**Comment by Ddedalus** [comment]
I've shipped 0.2.0 with #5 . Have a look at the updated README and let me know if this suits your workflow.

**Comment by okelet** [comment]
Works great, thanks!
